tests/puppeteer/campaigns/functional/BO/shopParameters/contact/01_filterContacts.js

```diff
@@ -0,0 +1,89 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ContactsPage = require('@pages/BO/shopParameters/contact/index');
+// Importing data
+const {Contacts} = require('@data/demo/contacts');
+
+let browser;
+let page;
+let numberOfContacts = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    contactsPage: new ContactsPage(page),
+  };
+};
+
+// Filter Contacts
+describe('Filter Contacts', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
```

Comment:
Not sure about this function, should 'this' be bind to this function context or a parent one ? Doubting about using a function instead of arrow function